drive.go

```diff
@@ -0,0 +1,115 @@
+package deta
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+)
+
+const (
+	uploadChunkSize = 1024 * 1024 * 10
+)
+
+var (
+	// ErrEmptyName empty name 
+	ErrEmptyName    = errors.New("name is empty")
+	// ErrEmptyNames empty names 
+	ErrEmptyNames   = errors.New("names is empty")
+	// ErrTooManyNames too many items 
+	ErrTooManyNames = errors.New("more than 1000 files to delete")
```

Comment:
the error string should not be `more than 100 files to delete` as it is specific to the `delete` function where as `ErrTooManyNames` is a general error